.circleci/config.yml

```diff
@@ -205,7 +205,9 @@ jobs:
       - *copy_generated_sql
       - run:
           name: Generate DAGs
-          command: PATH="venv/bin:$PATH" script/generate_airflow_dags
+          command: |
+            PATH="venv/bin:$PATH" script/generate_airflow_dags
+            mv dags/ /tmp/workspace/generated-sql
```

Comment:
DAGs get written to the `dags/` directory, but we persist `/tmp/workspace/generated-sql`, so we need to copy the generated DAGs to the right place